### PR TITLE
Adjust1-102 Delete tagged bail adjustment

### PR DESCRIPTION
**Pull request by pavilionsahota**

### CODE REVIEW

File: server/routes/taggedBailRoutes.ts [hunk 1]
```diff
@@ -103,6 +105,34 @@ export default class TaggedBailRoutes {
...
+    if (!adjustmentType) {
+      return res.redirect(`/${nomsId}`)
+    }
+
+    const sentencesAndOffences = await this.prisonerService.getSentencesAndOffences(prisonerDetail.bookingId, token)
+    const sentencesByCaseSequence = getActiveSentencesByCaseSequence(sentencesAndOffences)
+    const sentencesForCaseSequence = sentencesByCaseSequence.find(
+      it => it.caseSequence === adjustment.taggedBail.caseSequence,
+    )
+    const sentenceAndOffence = sentencesForCaseSequence.sentences.sort(
```
Comment (ldlharper) [inline server/routes/taggedBailRoutes.ts:127]: I think this sort function could be refactored somewhere. It appears a lot through all the tagged bail journeys. (Similar to getActiveSentencesByCaseSequence)

File: server/model/taggedBailRemoveModel.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { Adjustment } from '../@types/adjustments/adjustmentsTypes'
+import { AdjustmentType } from './adjustmentTypes'
+import { dateToString } from '../utils/utils'
+
+export default class TaggedBailRemoveModel {
+  constructor(
+    public prisonerDetail: PrisonApiPrisoner,
+    public adjustment: Adjustment,
+    public adjustmentType: AdjustmentType,
+    public sentenceAndOffense: PrisonApiOffenderSentenceAndOffences,
```
Comment (ldlharper) [inline server/model/taggedBailRemoveModel.ts:11]: minor spelling mistake should be `sentenceAndOffence`

**Review by ldlharper — COMMENTED** [review]
A couple of minor comments.

File: server/routes/taggedBailRoutes.ts (hunk 1 above)
Comment (pavilionsahota) [reply server/routes/taggedBailRoutes.ts]: I've created the method `getMostRecentSentenceAndOffence(sentencesAndOffences: PrisonApiOffenderSentenceAndOffences[])` in `utils.ts` and have replaced all usages of the sort with the call to this new utility method 👍

File: server/model/taggedBailRemoveModel.ts (hunk 2 above)
Comment (pavilionsahota) [reply server/model/taggedBailRemoveModel.ts]: I have corrected the spelling for this 👍

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**